spec/lib/object_serializer_class_methods_spec.rb

```diff
@@ -87,6 +87,28 @@
     end
   end
 
+  describe '#has_many with block and id_method_name' do
+    before do
+      MovieSerializer.has_many(:awards, id_method_name: :imdb_award_id) do |movie|
+        movie.actors.map(&:awards).flatten
+      end
+    end
+
+    after do
+      MovieSerializer.relationships_to_serialize.delete(:awards)
+    end
+
+    context 'awards is not included' do
+      subject(:hash) { MovieSerializer.new(movie).serializable_hash }
+
+      it 'returns correct hash' do
```

Comment:
from the test specification it is not clear that [or why] the hash is actually *correct*

is it possible to compare `hash[:data][:relationships][:awards][:data]` to a full hash where it will be visible that exactly `imdb_award_id` was used to serialize `id`

Reply:
Sure, I'll work on this!